COMETwebapp/COMETwebapp/Pages/Index.razor

```diff
@@ -22,7 +22,19 @@
 
 @page "/"
 @using COMETwebapp.Components
-
-<Home Title="Welcome to the COMET web application !" Color="#235180"/>
-
+@inject AuthenticationStateProvider AuthenticationStateProvider
+
+
+<div style="text-align:center">
+    <h1 style="color: #2e7191">
+        <AuthorizeView>
+            <Authorized>
+               Welcome @context.User.Identity.Name !
+            </Authorized>
+            <NotAuthorized>
+                Welcome to the COMET web application !
```

Comment:
for this and above no space before the !. I would also capitalize "Web Application"